var/spack/repos/builtin/packages/hypre/package.py

```diff
@@ -103,6 +104,9 @@ class Hypre(AutotoolsPackage, CudaPackage):
     # Option added in v2.16.0
     conflicts('+mixedint', when='@:2.15')
 
+    # HIP support added in v2.21.0
+    conflicts('+rocm', when='@:2.21.0')
```

Comment:
Given HIP support added in v2.21.0, you'll want to change the `when` clause here:

```suggestion
    conflicts('+rocm', when='@:2.20')
```

Reply:
Thanks @tldahlgren for the catch!